features/unit/transactions.feature

```diff
@@ -10,32 +10,32 @@ Feature: Transaction encoding
-      | closeout  | 100            | BH55E5RMBD4GYWXGX5W5PJ5JAHPGM5OXKDQH5DC4O2MGI7NW4H6VOE4CP4 |                                   |                                   | 0            | 0           | 0           | 0          | str:test | 5555,6666    | 7777,8888      | AAVDEAJ3NIYOG7XCRBKCJ3T5PUCVL2XASOP3NGX4NPPZ3UX6477PBG6E4Q,AADQIC4PMKRTFMHAAXYAFSGAUULDI2ABBIYVQJ6GZ5JHY6DJPHTU2SPHYM | 1234 | 9000        | 9010       | Mf0h6zjkEIEZPtNM3zsrg+iHQFS0fZxhgr7w35I464M= | gqNzaWfEQCgyXFm+653eOTGcswb0OPkF2NZe0nSZhSKBJzK+woWbo9dcl111qiR2xk3X0m21eTNynztPjvmloDKI10j/kAyjdHhujKRhcGFhkcQEdGVzdKRhcGFuAqRhcGFzks0eYc0iuKRhcGF0ksQgACoyATtqMON+4ohUJO59fQVV6uCTn7aa/GvfndL+5/7EIAAHBAuPYqMysOAF8ALIwKUWNGgBCjFYJ8bPUnx4aXnnpGFwZmGSzRWzzRoKpGFwaWRko2ZlZc0E0qJmds0jKKJnaMQgMf0h6zjkEIEZPtNM3zsrg+iHQFS0fZxhgr7w35I464OibHbNIzKjc25kxCAJ+9J2LAj4bFrmv23Xp6kB3mZ111Dgfoxcdphkfbbh/aR0eXBlpGFwcGw=                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
```

Comment:
Ahhh - here it is! It might be a bit easier to grok these pure box changes if you put the new box data either at the beginning or the end, or create brand new test cases at the bottom. 

For an even clearer diff, consider copy-pasta'ing the original test leaving the it untouched, and just add the box fields to that test case.